metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -347,12 +352,35 @@ def _make_bigquery_view(view: bigquery.Row) -> BigqueryView:
             rows_count=view.get("row_count"),
         )
 
+    def get_policy_tags_for_column(
```

Comment:
Please add error handling around this method. We don't want the whole ingestion to fail on
- error 
- rate limiting
- logging in the bigquery source report for such errors

Reply:
1. Implemented error handling to prevent the entire ingestion process from failing.
2. Incorporated rate limiting for the policy tags API.
3. add logging in the bigquery source report for such errors